tests/unit/EventSubscriberTest.php

```
@@ -44,4 +49,39 @@ public function testRegistersPostUpdateCmd(): void
             'Event subscriber does not register for the post-update-cmd event.',
         );
     }
+
+    public function testRunScaffoldAbortsWhenVendorDirIsEmpty(): void
+    {
+        $io = new BufferIO();
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $io. Configured minimum length is 3. (undefined)

(ShortVariable)